flow360/component/simulation/operating_condition/operating_condition.py

```diff
@@ -401,21 +404,34 @@ def operating_condition_from_mach_reynolds(
     ...     alpha=2.0 * u.deg,
     ...     beta=0.0 * u.deg,
     ...     reference_mach=0.85,
+    ...     project_length_unit=1 * u.mm,
     ... )
     >>> print(condition)
     AerospaceCondition(...)
 
     """
 
-    thermal_state = ThermalState(
-        temperature=temperature,
-        material=Air(
-            dynamic_viscosity=Sutherland(
-                reference_temperature=temperature,
-                reference_viscosity=(mach / reynolds) * u.flow360_viscosity_unit,
-                effective_temperature=110.4 * u.K,
-            )
-        ),
+    if temperature == 288.15 * u.K:
+        log.info("Default value of 288.15 K will be used as temperature.")
+
+    material = Air(
```

Comment:
You can just use default Air(). The rest looks good to me.

Reply:
True, and reference_temperature in Sutherland should be a default value instead of temperature variable. Thanks!